SumoLogic.Logging.Common/SumoLogic.Logging.Common.csproj

```diff
@@ -16,20 +16,15 @@
         <RepositoryType>git</RepositoryType>
         <SignAssembly>true</SignAssembly>
     </PropertyGroup>
-    <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' ">
-        <DefineConstants>netfull</DefineConstants>
-    </PropertyGroup>
-    <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-        <DefineConstants>netstandard</DefineConstants>
-    </PropertyGroup>
     <ItemGroup>
-        <None Include="..\SumoLogic.Logging.snk">
-            <Link>SumoLogic.Logging.snk</Link>
-        </None>
-        <CodeAnalysisDictionary Include="..\CustomDictionary.xml">
-            <Link>CustomDictionary.xml</Link>
-        </CodeAnalysisDictionary>
-        <PackageReference Include="System.Net.Http" Version="4.3.1" />
-        <PackageReference Include="System.Net.Requests" Version="4.3.0" />
+      <None Include="..\SumoLogic.Logging.snk">
+          <Link>SumoLogic.Logging.snk</Link>
+      </None>
+      <CodeAnalysisDictionary Include="..\CustomDictionary.xml">
+          <Link>CustomDictionary.xml</Link>
+      </CodeAnalysisDictionary>
+    </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' != 'netstandard2.0' ">
+      <PackageReference Include="System.Net.Http" Version="4.3.1" />

```

Comment:
Tab spacing seems to have changed - can we please revert back?

Reply:
Have not changed tab/spaces it is just github-comparison getting confused and not working well with xml. VS2019-comparison looks correct.